src/common/EvalIFrame.res

```diff
@@ -13,8 +13,9 @@ let srcDoc = `
         <title>Playground Output</title>
         <style>${css}</style>
       </head>
-      <body>
+      <body style="background-color:white;">
         <div id="root"></div>
+        <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
Another alternative is to simply force the white background using inline CSS, without loading Tailwind.